CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
+# react-components 5.26.3 (2021-01-29)
+
+- [Icon] Add Puppet Connect logo (by [@mardotio](https://github.com/mardotio) in [#379](https://github.com/puppetlabs/design-system/pull/379))
```

Comment:
I think this was supposed to be `Add inventory icon`?

Reply:
Thanks @mardotio! Updated.